sceptre/cli/helpers.py

```diff
@@ -173,6 +189,23 @@ def _generate_text(stream):
     return stream
 
 
+def process_template(
```

Comment:
While `process_template` as a nested function inside your other functions was a perfectly acceptable name, since you would contextually understand what "processing" meant. However, since you've extracted this function out and put it in `helpers`, it isn't obvious at all what "processing" means. I think we should have a better name on this function that clearly describes what it is supposed to do. Maybe `dump_dict_to_file`?

Reply:
fixed